ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+---
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
```

Comment:
It might be worth adding a note here that including a sample Workflow – if applicable – is the best way to show how to reproduce. Users sometimes describe behavior and don't include a workflow.

Reply:
I've added a checklist. People like to fill out checklists it seems.